apps/passport-client/new-components/shared/utils.ts

```diff
@@ -40,15 +40,15 @@ export const isMobile = /Mobi|Android/i.test(navigator.userAgent);
 
 export const useOrientation = (): ScreenOrientation => {
   const [state, setState] = useState<ScreenOrientation>(
-    window.screen.orientation
+    window.screen.orientation ?? { type: "portrait-primary" }
```

Comment:
nit: maybe we should do `window.screen?.orientation` for the 3% of devices that don't support it 
<img width="1365" alt="Screenshot 2024-11-10 at 15 20 55" src="https://github.com/user-attachments/assets/eb7d2cad-f21f-433a-9675-22d25fece28d">

Reply:
thanks @17Amir17